### PR TITLE
OSDOCS-11329: 4.16.3 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.16

Issue:
https://issues.redhat.com/browse/OSDOCS-11329

Link to docs preview:
https://78863--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-3_release-notes

QE review:
N/A

Additional information:
Errata URLs will return 404 until go-live (7/16)

### CODE REVIEW

**Comment by jeana-redhat** [comment]
cherrypick that contains the two machine mgmt xrefs (will merge when built): #78953

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3313,7 +3313,66 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, if a new deployment was performed at the OSTree level on a host that was identical to the current deployment but on a different stateroot, the OSTree saw them as equal. This behavior incorrectly prevented the bootloader from updating when `set-default` was invoked, as OSTree did not recognize the two stateroots as a differentiation factor for deployments. With this release, OSTree's logic has been modified to consider the stateroots and allows OSTree to properly set the default deployment to a new deployment with different stateroots. (link:https://issues.redhat.com/browse/OCPBUGS-36386[*OCPBUGS-36386*])
```
Comment (tmalove) [inline release_notes/ocp-4-16-release-notes.adoc:3362]:
```suggestion
* Previously, if a new deployment was performed at the OSTree level on a host that was identical to the current deployment but on a different stateroot, the OSTree saw them as equal. This behavior incorrectly prevented the boot loader from updating when `set-default` was invoked, as OSTree did not recognize the two stateroots as a differentiation factor for deployments. With this release, OSTree's logic has been modified to consider the stateroots and allows OSTree to properly set the default deployment to a new deployment with different stateroots. (link:https://issues.redhat.com/browse/OCPBUGS-36386[*OCPBUGS-36386*])
```

**Review by tmalove — APPROVED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -3316,7 +3316,66 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, if a new deployment was performed at the OSTree level on a host that was identical to the current deployment but on a different stateroot, the OSTree saw them as equal. This behavior incorrectly prevented the boot loader from updating when `set-default` was invoked, as OSTree did not recognize the two stateroots as a differentiation factor for deployments. With this release, OSTree's logic has been modified to consider the stateroots and allows OSTree to properly set the default deployment to a new deployment with different stateroots. (link:https://issues.redhat.com/browse/OCPBUGS-36386[*OCPBUGS-36386*])
```
Comment (cbippley) [inline release_notes/ocp-4-16-release-notes.adoc:3365]: This is the wrong link for this bug.

**Pull request merged**